app/ts/ru_RU.ts

```diff
@@ -614,253 +617,253 @@ Length: %4</source>
 <context>
     <name>olive::Core</name>
     <message>
-        <location filename="../core.cpp" line="213"/>
+        <location filename="../core.cpp" line="212"/>
         <source>Import error</source>
-        <translation type="unfinished"></translation>
+        <translation>Ошибка импорта</translation>
     </message>
     <message>
-        <location filename="../core.cpp" line="213"/>
+        <location filename="../core.cpp" line="212"/>
         <source>Nothing to import</source>
-        <translation type="unfinished"></translation>
+        <translation>Нет импортируемых данных</translation>
     </message>
     <message>
-        <location filename="../core.cpp" line="225"/>
+        <location filename="../core.cpp" line="224"/>
         <source>Importing...</source>
-        <translation type="unfinished"></translation>
+        <translation>Выполняется импорт…</translation>
     </message>
     <message>
-        <location filename="../core.cpp" line="321"/>
+        <location filename="../core.cpp" line="309"/>
         <source>Import footage...</source>
-        <translation type="unfinished"></translation>
+        <translation>Импортировать видеоматериал…</translation>
```

Comment:
While ellipsis is typografically correct, the source string doesn't use it.
```suggestion
        <translation>Импортировать видеоматериал...</translation>
```